include/ardpolyfills/Servo.h

```diff
@@ -16,19 +16,14 @@
  *
  */
 
-#ifndef SMARTCAR_EMUL_Servo_h
-#define SMARTCAR_EMUL_Servo_h
+#ifndef SMARTCAR_EMUL_SERVO_H
+#define SMARTCAR_EMUL_SERVO_H
 
 #include <cstdint>
 
-#define MIN_PULSE_WIDTH      1000
-#define MAX_PULSE_WIDTH      2000
-#define DEFAULT_PULSE_WIDTH  1500
+enum boundary{ MinPulseWidth = 1000, MaxPulseWidth = 2000, DefaultPulseWidth = 1500, InvalidServo = 255 };
```

Comment:
Enums are types; types should be in PascalCase, therefore, enums should be in PascalCase
Enumerators are constants; by Arduino convention, constants should be in SHOUTING_SNAKE_CASE

Reply:
CLion rename it for me as its recommendations

Reply:
That's not how identifiers casing conventions work
You're the human, you should know what to do; don't blindly follow what your unconfigured IDE is telling you